website/static/js/citations.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var $ = require('jquery');
+var $osf = require('osfHelpers');
+require('select2');
+require('../css/citations.css');
+
+var r = function(query) {
+    query.callback({results: [
+        {
+            _id: "academy-of-management-review",
+            summary: null,
+            short_title: "AMR",
+            title: "Academy of Management Review"
+        }
+    ]})
+}
+
+var formatResult = function(state) {
+    return "<div class='citation-result-title'>" + state.title + "</div>";;
+};
+
+var formatSelection = function(state) {
+    return state.title;
+};
+
+var input = $('#citationStyleInput');
```

Comment:
If the node is viewed through an anonymized VOL, these selectors will not exist. https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/project/project.mako#L194